lib/batcher.js

```diff
@@ -1,34 +1,34 @@
-
-
 function createBatcher(handleBatch, maxSize, maxTime) {
   return {
     dataArray: [],
     // using closure, so no need to keep as part of the object.
     // maxSize: maxSize,
     // maxTime: maxTime,
-    add: function(data) {
+    /** @type {any} */
```

Comment:
This type hint is needed to avoid ```TS9005: Declaration emit for this file requires using private name 'add'. An explicit type annotation may unblock declaration emit.```